ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -29,6 +24,27 @@ fn deserialise_string(v: Vec<u8>) -> Result<String, DekuError> {
     }
 }
 
+/// A trait for converting in-memory data-structures into a human-readable textual format.
+///
+/// This is modelled on [`std::fmt::Display`], but a reference to the module is always passed down
+/// so that constructs that require lookups into the module's tables from stringification have
+/// access to them.
+///
+/// The way we implement this (returning a `String`) is inefficient, but it doesn't hugely matter,
+/// as the human-readable format is only provided as a debugging aid, and isn't used in
```

Comment:
Remove ", and isn't used in 'production'" IMHO.

Reply:
Fixed in 20b53e05b6db8e1a775d99eb084c9be36505016a